src/main/java/net/minecraftforge/common/ForgeInternalHandler.java

```diff
@@ -45,25 +45,7 @@ public void onEntityJoinWorld(EntityJoinWorldEvent event)
         if (entity.getClass().equals(EntityItem.class))
         {
             ItemStack stack = ((EntityItem)entity).getEntityItem();
-
-            if (stack == null)
-            {
-                //entity.setDead();
-                //event.setCanceled(true);
-                return;
-            }
-
             Item item = stack.getItem();
-            if (item == null)
```

Comment:
*could* be false if its air, this is to protect against stupid modders, I have seen them screw this up.